monGARS/core/conversation.py

```
@@ -1,23 +1,28 @@
 import asyncio
 import logging
 from datetime import datetime
+
+from sqlalchemy import desc, select, update
```

Ruff (0.12.2)

`sqlalchemy.update` imported but unused

Remove unused import: `sqlalchemy.update`

(F401)